src/plugins/discover/public/components/data_types/logs/service_name_badge_cell.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import type { CSSObject } from '@emotion/react';
+import type { AgentName } from '@kbn/elastic-agent-utils';
+import { dynamic } from '@kbn/shared-ux-utility';
+import type { DataGridCellValueElementProps } from '@kbn/unified-data-table';
+import React from 'react';
+import { AGENT_NAME_FIELD } from '../../../../common/data_types/logs/constants';
+
+const dataTestSubj = 'serviceNameBadgeCell';
+const badgeCss: CSSObject = { marginTop: '-4px' };
```

Comment:
> [!NOTE]
> I guess this comes from a copy/paste as we don't have any badge in this component, can you please update the naming accordingly? Also, is this margin required since it's mostly text content in the flex group?